lib/tasks/test_replication.rake

```diff
@@ -1,4 +1,5 @@
 require_relative "./evm_test_helper"
+require "English"
```

Comment:
Yeah, I had to require it locally in irb, but it seems to work in the context of the rake task.  I'll take it out.